src/pages/docs/octopus-rest-api/octopus-cli/index.mdx

```diff
@@ -12,45 +12,47 @@ import OctoAutocompleteTabCompletion from 'src/shared-content/octopus-cli/octo-a
 
 <OctopusCli />
 
-## Commands \{#octoCommandLine-Commands}
+## Commands {#octoCommandLine-Commands}
```

Comment:
@steve-fenton-octopus I notice that the utility has removed the `\` which was in front of the `{` character.

I'm not familiar with our CMS... Is this a bug, or does it not matter?

Reply:
Good spot - this is going to be annoying, so prepare yourself.

In an MDX file, the `{` character get's interpreted as meaningful, so it needs to be escaped `\{`.

In an MD file, it shouldn't be escaped.

This is a really annoying issue with `.mdx`.

If it makes it simpler, we could make all the files MDX so you don't need to handle differences.

Reply:
I've done a commit to show the areas that need an update, hopefully it's a useful reference:

https://github.com/OctopusDeploy/docs/pull/1943/commits/fc5397d01641498aa5ea57c69b1b8f3d0d47225a

Reply:
Thanks steve. It's pretty simple to make the utility aware of mdx files and escaping the `{` so I'll go with that. I'll also fix up the paths